content/applications/productivity/mail_plugins/outlook.rst

```diff
@@ -75,6 +75,21 @@ Connect your database
       Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
       trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
 
+   .. tip::
+      If, after a short while, the panel is still empty, it is possible that your browser cookie
+      settings prevented it from loading.
+      Note that these settings also change if you are in "Incognito" mode on your
+      browser.
```

Comment:
Nitpicking for your future PRs: It's better not to return to a new line if you don't intend to start a new paragraph. Indeed, single line breaks have no effect on the rendered HTML so it can be confusing for the technical reader to see useless line breaks.